rskj-core/src/test/java/co/rsk/net/handler/txvalidator/TxValidatorNonceRangeValidatorTest.java

```diff
@@ -29,32 +29,51 @@
 public class TxValidatorNonceRangeValidatorTest {
 
     @Test
-    public void nonceInRange() {
+    public void oneSlotRange() {
         Transaction tx1 = Mockito.mock(Transaction.class);
         Transaction tx2 = Mockito.mock(Transaction.class);
+        Transaction tx3 = Mockito.mock(Transaction.class);
         AccountState as = Mockito.mock(AccountState.class);
 
         Mockito.when(tx1.getNonceAsInteger()).thenReturn(BigInteger.valueOf(0));
-        Mockito.when(tx2.getNonceAsInteger()).thenReturn(BigInteger.valueOf(3));
-        Mockito.when(as.getNonce()).thenReturn(BigInteger.valueOf(0));
+        Mockito.when(tx2.getNonceAsInteger()).thenReturn(BigInteger.valueOf(1));
+        Mockito.when(tx3.getNonceAsInteger()).thenReturn(BigInteger.valueOf(2));
+        Mockito.when(as.getNonce()).thenReturn(BigInteger.valueOf(1));
 
-        TxValidatorNonceRangeValidator tvnrv = new TxValidatorNonceRangeValidator();
-        Assert.assertTrue(tvnrv.validate(tx1, as, null, null, Long.MAX_VALUE, false).transactionIsValid());
+        TxValidatorNonceRangeValidator tvnrv = new TxValidatorNonceRangeValidator(1);
+        Assert.assertFalse(tvnrv.validate(tx1, as, null, null, Long.MAX_VALUE, false).transactionIsValid());
         Assert.assertTrue(tvnrv.validate(tx2, as, null, null, Long.MAX_VALUE, false).transactionIsValid());
+        Assert.assertFalse(tvnrv.validate(tx3, as, null, null, Long.MAX_VALUE, false).transactionIsValid());
     }
 
     @Test
-    public void nonceOutOfRange() {
+    public void fiveSlotsRange() {
```

Comment:
When testing i would like to know what happens in the border cases, for instance:
- nonce = config -1 / config / config +1 / 0 / high

Reply:
I guess these cases are covered in the `oneSlotRange` and `fiveSlotsRange` tests. Or did I miss something?

- when accountSlots == 1 and account nonce == 1 and tx nonce is [0, 1, 2]
- when  accountSlots == 5 and account nonce == 1 and tx nonce is [0, 1, 5, 6]

Reply:
Ok, i see. The only one left is the 2nd case then, when nonce is 4, right?